src/main/kotlin/org/wycliffeassociates/tstudio2rc/Converter.kt

```diff
@@ -0,0 +1,149 @@
+package org.wycliffeassociates.tstudio2rc
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import org.wycliffeassociates.resourcecontainer.entity.Manifest
+import org.wycliffeassociates.tstudio2rc.TextToUSFM.Companion.getVersification
+import java.io.File
+import java.nio.file.Files
+import java.util.zip.ZipFile
+
+/**
+ * API for converting a tstudio project to resource container(s).
+ */
+class Converter {
```

Comment:
This whole thing could just be an object, or just a file with functions, because there isn't anything stateful- you're not using the object and passing it around, it's just housing utilities